addons/mail/static/src/core/common/message_reactions.xml

```diff
@@ -10,12 +10,13 @@
         <button t-foreach="props.message.reactions" t-as="reaction" t-key="reaction.content" class="o-mail-MessageReaction btn d-flex p-0 border rounded-1 mb-1"
             t-on-click="() => this.onClickReaction(reaction)"
             t-on-contextmenu="onContextMenu"
+            t-on-mouseover="(ev) => this.openReactions({ev: ev, reaction: reaction})"
```

Comment:
`{ev: ev, reaction: reaction}` => `{ev, reaction}` (shorter)

I would also rename function as `onMouseoverReaction`, so it's much easier to track function from where this is called. I get that imperative naming of function looks easier at first, but it's actually harder to track jumps in the code.